src/ucp/wireup/select.c

```diff
@@ -1196,7 +1195,29 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
     bw_info.max_lanes         = context->config.ext.max_rndv_lanes;
     bw_info.usage             = UCP_WIREUP_LANE_USAGE_RMA_BW;
 
+    /* check rkey_ptr */
+    if (!(ep_init_flags & UCP_EP_INIT_FLAG_MEM_TYPE) &&
+         (context->config.ext.rndv_mode == UCP_RNDV_MODE_AUTO)) {
+
+        /* We require remote memory registration and local ability to obtain
+         * a pointer to the remote key. Only one is needed since we are doing
+         * memory copy on the CPU.
+         * Allow selecting additional lanes in case the remote memory will not be
+         * registered with this memory domain, i.e wit GPU memory.
+         */
+        bw_info.criteria.remote_md_flags = md_reg_flag;
```

Comment:
can set `bw_info.criteria.remote_md_flags` only once before this `if` statement
and remove it from 1208 and 1218 lines

Reply:
done